utah-design-system-website/src/react/components/routing/Routing.jsx

```diff
@@ -73,8 +73,13 @@ export function Routing() {
               <DocumentationTemplate
                 content={page.content}
                 contentRef={contentRef}
-                // @ts-ignore
-                sidePanelLeftContent={menuSecondary && <VerticalMenu currentMenuItem={currentMenuItem} menus={menuSecondary} />}
+                sidePanelLeftContent={menuSecondary && (
+                  <VerticalMenu
+                    className="menu-side-panel"
+                    currentMenuItem={currentMenuItem}
+                    menus={menuSecondary ?? null}
```

Comment:
Still getting an error
`Type  undefined  is not assignable to type  WebsiteMainMenu`

Reply:
:(